### DISCUSSION TITLE
怎么添加频道？

### DISCUSSION THREAD

**Discussion by fanmlei** [chanify/chanify#98] [Q&A]
在app里面只有几个默认的频道，没有找到添加的入口，请问可以添加自定义的频道吗

**Comment by liyuan5087** [comment]
长按扫描二维码可以添加频道

**Reply by wizjin** [reply]
@fanmlei 你好，长按在首页右上角的扫描二维码按钮，会弹出菜单，里面有添加频道的选项。

@liyuan5087 谢谢帮忙回答。

**Reply by fanmlei** [reply]
谢谢

**Reply by fanmlei** [reply]
> @fanmlei 你好，长按在首页右上角的扫描二维码按钮，会弹出菜单，里面有添加频道的选项。
> 
> @liyuan5087 谢谢帮忙回答。

感觉入口隐藏的有点深，找了好久没找到。。。

**Comment by daveteu** [comment]
what is `Code` for?

![IMG_4094](https://user-images.githubusercontent.com/18649392/147755351-1c6e68c9-f97c-43fe-840a-aac05d4117a2.jpg)

**Reply by wizjin** [reply]
`Code` is identify for channel.
